src/WebJobs.Extensions.DurableTask/Microsoft.Azure.WebJobs.Extensions.DurableTask-net471.xml

```diff
@@ -508,10 +508,23 @@
             <param name="timeout">Total allowed timeout for output from the durable function. The default value is 10 seconds.</param>
             <returns>An HTTP response which may include a 202 and location header or a 200 with the durable function output in the response body.</returns>
         </member>
+<<<<<<< HEAD
```

Comment:
These merge conflicts need to be resolved

Reply:
@amdeel, the easiest way to resolve these is to just rebuild the project once all of your *.cs merge conflicts are resolved.